integration_tests/__tests__/__snapshots__/custom_matcher.test.js.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Custom matcher preserves error stack 1`] = `
+"Error: qux
+    at baz (/Users/bvaughn/Documents/git/jest/integration_tests/__tests__/custom_matcher.test.js:49:13)
```

Comment:
this shouldn't happen, right? we should still clean it up to be relative paths (especially for this integration test, but in general as well)

Reply:
Or do we really not want prettified stack trace at all?

Reply:
😆  Sorry. This was super silly of me. I'm embarrassed. I noticed that CI failed but hadn't had a chance yet to circle back and see why.

Reply:
I'm not sure if I fixed this in the correct way. 😄